netsim/ansible/tasks/linux/initial-clab.yml

```diff
@@ -25,13 +25,9 @@
 - name: "Initial container configuration via {{ exec_script }}"
   shell: |
     set -e
-    pid=$(docker inspect -f {% raw %}'{{.State.Pid}}'{% endraw %} clab-{{ netlab_name }}-{{inventory_hostname }})
-    mkdir -p /var/run/netns
-    ln -s /proc/$pid/ns/net /var/run/netns/$pid
-    ip netns exec $pid bash {{ exec_script }}
-    rm /var/run/netns/$pid
+    ip netns exec clab-{{ netlab_name }}-{{ inventory_hostname }} bash {{ exec_script }}
```

Comment:
I looked into changing the ```prefix``` in Containerlab to get 'clean' DNS names for the nodes such that name resolution would work the way it does in the physical network, but this "clab-xxx" naming convention is hard-coded in too many places. Hence the PR for DNS support in clab such that I could point at that instead

Still, I think we should set ```netlab_container_name``` or something similar at some right place in the script, and use that everywhere instead (such that we might change it easier in the future)

Reply:
I'm assuming you wanted to get rid of the `clab-` prefix, not changing `netlab-name` (because that should be easy to do with the **name** attribute).

That's way too much work, and we would never be absolutely sure whether we caught all the instances. I don't think it's worth the effort.